backend/graph/api/schema/mutations.graphqls

```diff
@@ -21,8 +21,17 @@ type MutationRoot {
 
     answerSurveyQuestion(id: UUID!, answer: String!): AnswerSurveyQuestionResult!
     updateSurveyQuestionAnswer(key: String!, answer: String!): AnswerSurveyQuestionResult!
+
+    addEpisodeToMyList(episodeId: ID!): AddToCollectionResult!
+    addShowToMyList(showId: ID!): AddToCollectionResult!
```

Comment:
Would make more sense to just return the `UserCollection`? 
Not sure how it will be used, maybe @andreasgan has some more opinion here?

Reply:
It depends on typing. Not sure yet if we want to return more metadata or if the result should be different. Having a type for it only slightly increases clutter, but simplifies additions in the future.